src/java/org/apache/cassandra/cql3/statements/UpdateStatement.java

```diff
@@ -348,4 +365,26 @@ public AuditLogContext getAuditLogContext()
     {
         return new AuditLogContext(AuditLogEntryType.UPDATE, keyspace(), table());
     }
+
+    public static void evaluateConstraintsForRow(Row row, TableMetadata metadata)
+    {
+
+        for (ColumnMetadata column : metadata.columnsWithConstraints)
+        {
+            Cell<?> cell = row.getCell(column);
```

Comment:
@bbotella `getCell`  can return `null`. That would happen when a row to insert / update does not contain the cell of given column. You can have a row consisting of less columns / cells to update than total number of columns in a table.

Reply:
I see. Let me check for null then.